src/main/resources/data/minecolonies/recipes/iron_nugget_from_iron_scimitar_blasting.json

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "minecraft:blasting",
+  "ingredient": [
+    {
+      "item": "minecolonies:iron_scimitar"
+    }
+  ],
+  "result": "minecraft:iron_nugget",
+  "experience": 0.1,
+  "cookingtime": 200

```

Comment:
This should be cookingtime 100.  Blasting is supposed to be twice as fast as smelting, and this is what iron swords do.

Reply:
ravenderps++

hotfix PR incoming 😳